src/app/shared/components/guided-tour/steps/add-project-btn/component.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component} from '@angular/core';
+import {DialogRef} from '../../dialog/overlay-ref';
+import {DialogHelperService} from '../../dialog/helper.service';
+
+@Component({
+  selector: 'km-gt-add-project-btn',
+  templateUrl: 'template.html',
+})
+export class GTAddProjectBtnComponent {
```

Comment:
For the most part, we should actually have a generic `GuidedTourStep` base component that could just offer a simple text display. It could be extended only if we'd need to display a very specific/custom step.

Reply:
But wasn't that the case previously, with all information in one file (sure, we wanted to get rid of that) and one component where it was applied to? At least very basic? Seems like I understand your idea differently than you.